### PR TITLE
Fix telco5g e2e

### PR DESCRIPTION
**Pull request by sabinaaledort**
1. Skip pre step in kcli plan
2. Bump go version to 1.17
3. Remove debugging sleep
4. Wait for kcli to create the cluster

### CODE REVIEW

**Comment by sabinaaledort** [comment]
/hold
just a temp PR till me fix https://github.com/openshift/release/pull/29217

File: ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-ref.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ ref:
   from_image:
     namespace: open-cluster-management
     name: builder
-    tag: go1.16-linux
+    tag: go1.17-linux
```
Comment (yanjoseph) [inline ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-ref.yaml:6]: As cnf-tests has already switched to go 1.18 why not not switch to that ?

Comment (sabinaaledort) [reply ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-ref.yaml:6]: it doesn't exist yet for open-cluster-management/builder image we use, i tried it here https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/29558/rehearse-29558-periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g/1537408968882655232/build-log.txt

Comment (sabinaaledort) [reply ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-ref.yaml:6]: let's merge this one to get a working CI and I will open another pr to update the image

**Pull request merged**